nion/swift/DocumentController.py

```diff
@@ -3292,6 +3292,7 @@ def execute(self, context: Window.ActionContext) -> Window.ActionResult:
 Window.register_action(SetToolModeAction("wedge", _("Wedge"), "wedge_icon.png", _("Wedge tool for creating wedge masks")))
 Window.register_action(SetToolModeAction("ring", _("Ring"), "annular_ring.png", _("Ring tool for creating ring masks")))
 Window.register_action(SetToolModeAction("lattice", _("Lattice"), "lattice_icon.png", _("Lattice tool for creating periodic lattice masks")))
+Window.register_action(SetToolModeAction("zoom", _("Zoom"), "zoom.png", _("Zoom in/out on image")))
```

Comment:
I'm not sure of the perfect overall ordering since we now have an odd number of general tools (pointer, zoom, hand), and this might need rework in the future, but I think the best way is to put "zoom" as the 3rd item in the list, after "hand". That would put it at the top of the 2nd column in the tools. It should definitely not be the last item.

In addition, the tool window is no longer aligned when it has an odd number of items. I think this can be fixed easily with this patch, which should be included in this PR:

```
diff --git a/nion/swift/ToolbarPanel.py b/nion/swift/ToolbarPanel.py
index 1dd95410..a7fbf41d 100644
--- a/nion/swift/ToolbarPanel.py
+++ b/nion/swift/ToolbarPanel.py
@@ -67,6 +67,8 @@ class ToolModeToolbarWidget(Declarative.Handler):
                 top_row_items.append(radio_button)
             else:
                 bottom_row_items.append(radio_button)
+        if len(tool_actions) % 2 == 1:
+            bottom_row_items.append(u.create_spacing(32))
 
         top_row = u.create_row(*top_row_items)
         bottom_row = u.create_row(*bottom_row_items)
```